### DISCUSSION TITLE
i got a error

### DISCUSSION THREAD

**Discussion by Dev-RaviSwami** [python-telegram-bot/python-telegram-bot#3255] [Q&A] [answered]
Traceback (most recent call last):
  File "/home/darkdevil/Desktop/tg bot/Dazzy_Mark_Bot/bot.py", line 2, in <module>
    from telegram import Update
ImportError: cannot import name 'Update' from 'telegram' (/home/darkdevil/.local/lib/python3.10/site-packages/telegram/__init__.py)

but in my 2nd pc its run properly

**Comment by Bibo-Joshi** [comment, marked as answer]
Please have a look at [this faq entry](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#why-am-i-getting-importerror-cannot-import-name-xy-from-telegram).

**Reply by Dev-RaviSwami** [reply]
it sloved but another issu stand 

Traceback (most recent call last):
  File "/home/darkdevil/Desktop/tg bot/Dazzy_Mark_Bot/bot.py", line 3, in <module>
    from telegram.ext import ApplicationBuilder, CommandHandler, MessageHandler, ContextTypes
ImportError: cannot import name 'ApplicationBuilder' from 'telegram.ext' (/usr/local/lib/python3.10/dist-packages/telegram/ext/__init__.py)

**Reply by Bibo-Joshi** [reply]
Please have a look at #3040
